[Merged by Bors] - Removed frame rate updates from fixed step example

YohDeadfall opened the pull request:
Fixes #1751.

Output before:

```
...
update: 0.016478100000000495
update: 0.01955749999999945
fixed_update: 2.0052284
  overstep_percentage: 0.00946374999999966
update: 0.013902800000000326
update: 0.02283240000000042
...
```

Output after:

```
...
fixed_update: 2.0053731
  overstep_percentage: 0.0031439500000001175
fixed_update: 1.9976363
  overstep_percentage: 0.0019621000000002997
fixed_update: 2.0069121999999995
  overstep_percentage: 0.005418200000000262
...
```

alice-i-cecile reviewed (approved):
This keeps the example simpler, reduces spam and should be much less confusing.

mockersf commented:
I would rather have the log from frame update made clearer, so that it's obvious one is running with the frame and the other with the `FixedTimestep`

alice-i-cecile commented:
> I would rather have the log from frame update made clearer, so that it's obvious one is running with the frame and the other with the FixedTimestep

In order to do this, I think we need to have two different displays in some form, or to show only the "average frame time". You can't output it into the terminal raw or the fixed time step logging just gets swamped.

mockersf commented:
Or we could reduce the `FixedTimestep` from `step(2.0)` to `step(0.5)` which would make it happen more often, and with a log with a more obvious length difference

alice-i-cecile commented:
> Or we could reduce the FixedTimestep from step(2.0) to step(0.5) which would make it happen more often, and with a log with a more obvious length difference

Oh that's clever, and a very easy fix. Maybe that plus slightly clearer logging messages?

YohDeadfall commented:
Addressed review, but renamed `update` to `frame_update` for clarity, but left the message untouched because `frame_update` and `fixed_update` look very similar in a terminal.

cart commented:
bors r+
